distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -927,14 +953,6 @@ type File_Error
         Io_Error file msg -> msg.to_text + " (" + file.path + ")."
 
 
-## PRIVATE
-
-   Utility function that lists immediate children of a directory.
-list_immediate_children : File -> Vector.Vector File
-list_immediate_children directory =
-    arr = directory.prim_file.list
-    Vector.new (Polyglot.get_array_size arr) ix->
-        File (arr.at ix)
```

Comment:
Shouldn't we still keep a variant of this method to be used for users which returns a `Vector`?

A builtin returning an Array isn't something that should be user facing. Although I understand that this method was private to begin with, but I'm wondering if we couldn't make it public (if it returns vector)

Reply:
Added `list_immediate_children` and `list_immediate_children_array` since we were converting it to Vector anyway in other places. Good call